src/chttpd.erl

```diff
@@ -893,7 +893,7 @@ maybe_handle_error(Error) ->
         {Err, Reason} ->
             {500, couch_util:to_binary(Err), couch_util:to_binary(Reason)};
         Error ->
-            {500, couch_util:to_binary(Error), null}
+            {500, couch_util:to_binary({<<"unknown_error">>, Error}), null}
```

Comment:
noting that the original diff is;

 error_info(Error) ->
-    {500, <<"unknown_error">>, couch_util:to_binary(Error)}.
+    maybe_handle_error({<<"unknown_error">>, Error}).

So I think we want to match that in the Error case.